stable/traefik/templates/configmap.yaml

```diff
@@ -207,7 +207,7 @@ data:
     [zookeeper]
     endpoint = "{{ .Values.kvprovider.zookeeper.endpoint }}"
     watch = {{ .Values.kvprovider.zookeeper.watch }}
-    prefix = {{ .Values.kvprovider.zookeeper.prefix }}
+    prefix = "{{ .Values.kvprovider.zookeeper.prefix }}"
```

Comment:
```suggestion
    prefix = {{ .Values.kvprovider.zookeeper.prefix | quote }}
```

Reply:
Thanks dtomcej! I went without this function since it isn't used elsewhere in this file, e.g.:

`endpoint = "{{ .Values.kvprovider.zookeeper.endpoint }}"`

would you like me to clean it up to use the piped quote technique across the board, including adding the quotation to the prefix of other KV stores? If so I'm happy to make this change.

Reply:
That would be great if you could!